ros/src/twist_controller/dbw_node.py

```diff
@@ -45,6 +45,9 @@ def __init__(self):
         steer_ratio = rospy.get_param('~steer_ratio', 14.8)
         max_lat_accel = rospy.get_param('~max_lat_accel', 3.)
         max_steer_angle = rospy.get_param('~max_steer_angle', 8.)
+        throttle_gains = [rospy.get_param('~throttle_k_p', 0.5), rospy.get_param('~throttle_k_i', 0.00001),rospy.get_param('~throttle_k_d', 0.0)]
```

Comment:
I think it will be more readable if you don't use `rospy.get_param` inside of the list,
something like:
```python
throttle_kp = ospy.get_param('~throttle_k_p', 0.5)
throttle_ki = ospy.get_param('~throttle_k_i', 0.0001)
throttle_kd = ospy.get_param('~throttle_k_d', 0.0 )

throttle_gains = [throttle_kp, throttle_ki, throttle_kd]
```

Reply:
Will fix.